packages/cli/src/commands/doctor/types.ts

```diff
@@ -66,15 +66,42 @@ export type EnvironmentInfo = {
   };
 };
 
+export type HealthcheckCategory = {
+  label: string;
+  healthchecks: HealthCheckInterface[];
+};
+
+export type Healthchecks = {
+  common: HealthcheckCategory;
```

Comment:
Please rename to `HealthChecks` and `HealthCheckCategory` to stay consistent consistency